app/js/views/element_call_button_view.js

```diff
@@ -0,0 +1,54 @@
+import { View } from 'components/fxos-mvc/dist/mvc';
+import 'components/gaia-button/gaia-button';
+
+(function(window) {
+'use strict';
+
+var shadowTemplate =
+`<style scoped>
+gaia-button {
+	--button-background: green;
+	--button-background-active: lightgreen;
+	--button-color: white;
+	--button-color-active: white;
+	width: 10rem;
+	min-width: 0 !important;
+	margin: 0.5rem !important;
+}
+</style>
+<gaia-button>Call</gaia-button>`;
+
+var proto;
+
+export default class ElementCallButtonView extends View {
+	constructor(controller, phoneFieldController) {
+		proto = Object.create(HTMLElement.prototype);
+		proto.createdCallback = this._createdCallback;
+		proto._scope = this;
```

Comment:
At this point I'm probably just echoing the thoughts of Kevin Wilson and Justin, but I'll add my thoughts anyway.

Web components and Views are two different things, and the concepts should be kept separate. A view can have web components in it, but a web component doesn't needs it's own view or controller because we get the encapsulation already built in to web components.

Also, a view should have no concept of controllers, and should contain very little logic. All a view should worry about is the DOM content it owns (which can include web components and sub views), and propagating events associated with it's DOM. As a corollary, a controller shouldn't need to register event handlers with individual DOM elements in a view, but rather with the view itself. This makes it easier to change markup details without breaking the view/controller interaction.

Not every view needs it's own controller. Some views are just widgets or markup snippets that can be used by multiple controllers. An example that comes to mind is of a screen controller that, for instance, has a tab view, a list view, and a drawer view in it but doesn't necessarily need controllers for each one.

I feel like we only need to use web components when the encapsulation it provides is helpful. For most cases, a view and perhaps an associated controller is enough.